templates/shared-memory-service.json

```diff
@@ -12,7 +12,7 @@
          "openshift.io/display-name": "JBoss Data Grid Shared Memory Service",
          "openshift.io/provider-display-name": "Red Hat, Inc.",
          "openshift.io/documentation-url": "https://www.redhat.com/en/technologies/jboss-middleware/data-grid",
-         "openshift.io/long-description": "In this image, JBoss Data Grid is configured as a shared memory service with persistence. That means any data stored by the service will be preserved across restarts. Fault-tolerance is achieved by replicating data across multiple nodes.\nThe data is copied to a specific number of nodes in the cluster using the \"owners\" setting. This ensures that, in the case of failure, the grid will not lose data as long as the number of failed nodes is less than the number of copies (owners). This also works when nodes are shutdown or restarted manually.",
+         "openshift.io/long-description": "n this image, JBoss Data Grid is configured as a shared memory service with fault-tolerance. That means any data stored by the service will be preserved across restarts. fault-tolerance is achieved by replicating data across multiple nodes.\nThe data is copied to a specific number of nodes in the cluster using the \"owners\" setting. This ensures that, in the case of failure, the grid will not lose data as long as the number of failed nodes is less than the number of copies (owners). This also works when nodes are shutdown or restarted manually.",
```

Comment:
- I think we're missing the 'I' in "In"
- "fault-tolerance is achieved" should be "Fault-tolerance" ...

Never say never again :stuck_out_tongue_winking_eye:

Reply:
Fixed both. Update will arrive in a second.